CONTRIBUTING.md

```diff
@@ -2,9 +2,55 @@
 
 We want to make contributing to this project as easy and transparent as possible.
 
-## Our Development Process
+## TL;DR
 
-... (in particular how this is synced with internal changes to the project)
+We appreciate all contributions. If you are interested in contributing to TorchData, there are many ways to help out.
+Your contributions may fall into the following categories:
+
+- It helps the project if you can
+
+  - Report issues that you're facing
+  - Give a :+1: on issues that others reported and that are relevant to you
+
+- Answering questions on the issue tracke, investigating bugs are very valuable contributions to the project.
+
+- You would like to improve the documentation. This is no less important than improving the library itself! If you find
+  a typo in the documentation, do not hesitate to submit a GitHub pull request.
+
+- If you would like to fix a bug:
+
+  - comment on the issue that you want to work on this issue
+  - send a PR with your fix, see below.
+
+- If you plan to contribute new features, utility functions or extensions, please first open an issue and discuss the
+  feature with us.
+  - We have a checklist of things to go through while adding a new DataPipe. See below.
+- If you would like to feature a usage example in our documentation, discuss that with us in an issue.
+
+## Issues
+
+We use GitHub issues to track public bugs. Please follow the existing templates if possible and ensure that the
+description is clear and has sufficient instructions to be able to reproduce the issue.
+
+## Development installation
+
+### Install PyTorch Nightly
+
+```bash
+conda install pytorch -c pytorch-nightly
+# or with pip (see https://pytorch.org/get-started/locally/)
+# pip install numpy
+# pip install --pre torch -f https://download.pytorch.org/whl/nightly/cu102/torch_nightly.html
```

Comment:
Let's change it to `cpu` rather than `cuda` version to prevent misunderstanding about cuda dependency.